scripts/benchmark_stats.py

```diff
@@ -0,0 +1,60 @@
+from csv import DictReader
+from glob import glob
+from statistics import mean
+from typing import Generator, Iterable, List
+
+
+class BenchmarkStats:
+    def __init__(self, folder_path: str):
+        self._files: List[str] = glob(folder_path + "/*stats.csv")
+        self.get_requests: List[int] = []
+        self.post_requests: List[int] = []
+        self.total_requests: int = 0
+        self.total_failures: int = 0
+        self.error_percentage: int
+        self._data = self._process_file_data()
+
+    def __str__(self):
+        return (
+            f'Questionnaire GETs average: {int(self.average_get)}ms\n'
+            f'Questionnaire POSTs average: {int(self.average_post)}ms\n'
+            f'All requests average: {int(self.average_total)}ms\n'
+            f'Total Requests: {int(self.total_requests):,}\n'
+            f'Total Failures: {int(self.total_failures):,}\n'
+            f'Error Percentage: {(round(self.error_percentage, 2))}%\n'
+        )
+
+    def _process_file_data(self):
+        for file in self.files:
+            with open(file) as fp:
+                for row in DictReader(fp, delimiter=","):
+                    percentile_99th = int(row["99%"])
+                    if "/questionnaire" in row['Name']:
+                        if row["Type"] == "GET":
+                            self.get_requests.append(percentile_99th)
+                        elif row["Type"] == "POST":
+                            self.post_requests.append(percentile_99th)
+
+                    if row["Name"] == "Aggregated":
+                        self.total_requests += int(row["Request Count"])
+                        self.total_failures += int(row["Failure Count"])
+
```

Comment:
Need to handle old format, as existing files cannot be done using this. i.e Daily test stats will fail. We should look at archiving the current benchmark data in GCS if we didn't wsnt to handle the old data.